### PR TITLE
Develop - added buzz; a blog of sorts

### PR DESCRIPTION
**Pull request by armgitaar**
includes ability to manage (created, edit, delete) blog data

### CODE REVIEW

File: src/main/kotlin/providers/RouteServiceProvider.kt [hunk 1]
```diff
@@ -3,9 +3,10 @@ package dev.alpas.alpasdev.providers
 import dev.alpas.alpasdev.addRoutes
```
Comment (ashokgelal) [inline src/main/kotlin/providers/RouteServiceProvider.kt]: I think this file can be removed

**Comment by armgitaar** [comment]
Added validation guard, updated buzz design, updated alt tags on homepage for SEO

File: src/main/kotlin/controllers/BlogController.kt [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+        val createdAt = getDateTime(call.stringParam("createdAt"))
+
+        Blogs.create() {
+            it.title to call.stringParam("title")
+            it.url to call.stringParam("url")
+            it.createdAt to createdAt
+            it.body to call.stringParam("body")
+
+            call.redirect().back()
```
Comment (ashokgelal) [inline src/main/kotlin/controllers/BlogController.kt:51]: should be after flash

File: src/main/kotlin/controllers/BlogController.kt [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+
+        Blogs.update {
+            it.title to call.stringParam("title")
+            it.url to call.stringParam("url")
+            it.createdAt to createdAt
+            it.body to call.stringParam("body")
+
+            where{ it.id eq id }
+
+            call.redirect().toRouteNamed("blog")
```
Comment (ashokgelal) [inline src/main/kotlin/controllers/BlogController.kt:80]: After flash

File: src/main/kotlin/controllers/BlogController.kt [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+        }
+
+        flash("success", "Successfully Saved")
+    }
+
+    fun delete(call: HttpCall) {
+        val id = call.longParam("id").orAbort()
+        Blogs.delete { it.id eq id }
+
+        call.redirect().toRouteNamed("blog")
```
Comment (ashokgelal) [inline src/main/kotlin/controllers/BlogController.kt:88]: After flash

File: src/main/kotlin/controllers/BlogController.kt [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+
+        flash("success", "Successfully removed the stinger 🐝")
+    }
+}
+
+private fun getDateTime(date : String?): Instant? {
+    return LocalDateTime.parse(date, DateTimeFormatter.ofPattern("yyyy-MM-dd'T'HH:mm")).toInstant(ZoneOffset.ofHours(-6))
```
Comment (ashokgelal) [inline src/main/kotlin/controllers/BlogController.kt:95]: The offset maybe different on the deployed server that lives in a different timezone

File: src/main/kotlin/entities/User.kt [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+
+// https://alpas.dev/docs/ozone#dao
+interface User : BaseUser<User> {
+    // https://alpas.dev/docs/email-verification
+    override val mustVerifyEmail get() = true
+
+    companion object : OzoneEntity.Of<User>()
+
+    fun isAdmin(): Boolean {
+        return email in listOf("adam.miedema@gmail.com", "ashokgelal@gmail.com")
```
Comment (ashokgelal) [inline src/main/kotlin/entities/User.kt]: We should probably hide the email addresses

File: src/main/resources/templates/auth/emails/reset.peb [hunk 7]
```diff
@@ -0,0 +1,188 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
```
Comment (ashokgelal) [inline src/main/resources/templates/auth/emails/reset.peb:1]: Disable this?

File: src/main/resources/templates/auth/emails/verify.peb [hunk 8]
```diff
@@ -0,0 +1,187 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
```
Comment (ashokgelal) [inline src/main/resources/templates/auth/emails/verify.peb]: We can delete this

File: src/main/resources/templates/auth/passwords/reset.peb [hunk 9]
```diff
@@ -0,0 +1,58 @@
+{% extends "../../layout/app.peb" %}
```
Comment (ashokgelal) [inline src/main/resources/templates/auth/passwords/reset.peb:1]: We should hide this

File: src/main/resources/templates/auth/passwords/reset_email.peb [hunk 10]
```diff
@@ -0,0 +1,40 @@
+{% extends "../../layout/app.peb" %}
```
Comment (ashokgelal) [inline src/main/resources/templates/auth/passwords/reset_email.peb:1]: Hide this (and by this I mean password reset route)

File: src/main/resources/templates/home.peb [hunk 11]
```diff
@@ -1,84 +1,15 @@
 {% extends 'layout/app.peb' %}
 
-{% block title %} Delightful Web Development in Kotlin {% endblock %}
+{% block title %} The best blog in the universe {% endblock %}
```
Comment (ashokgelal) [inline src/main/resources/templates/home.peb]: Hmmmm... Let's not say that

File: src/main/resources/templates/layout/app.peb [hunk 12]
```diff
@@ -13,15 +13,45 @@
...
     {% block content %} {% endblock %}
 </div>
+
 {% include '_algolia_footer.peb' %}
+
+{% block pagescripts %} {% endblock %}
+
+<script>
+    $(document).ready(function() {
+        $('#flash').fadeOut(5000);
```
Comment (ashokgelal) [inline src/main/resources/templates/layout/app.peb]: Let's not add a full jQuery library for this one small function

File: src/main/kotlin/controllers/BlogController.kt (hunk 5 above)
Comment (armgitaar) [reply src/main/kotlin/controllers/BlogController.kt:95]: San Francisco?

**Pull request merged**